### ISSUE TITLE
How about core 3.1/net 5.0 and nuget?

### ISSUE THREAD

**Issue by SchreinerK** [mwinapi/mwinapi#10]
could we convert the project to SDK format and use multiple target frrameworks and also then publish to nuget?

**Comment by schierlm** [comment]
If you want to do that, feel free to send pull requests.

I myself do not longer write native Windows GUI applications, so I stopped following .NET development somewhere around .NET 4 (which also explains that this library is still built for .net 2.0 although my main development machine does not even have .NET 2/3 installed).

I also don't mind handing this project (or its enclosing GitHub organization and SourceForge project) over to new maintainers if they want to maintain this project further.

**Comment by schierlm** [comment]
Pinging @Aldaviva and @evilc as their forks seem to be the only ones with somewhat recent commit activity :)

**Comment by Aldaviva** [comment]
Yeah that's a good idea, I've been meaning to try and see if I can get that to work.

**Comment by SchreinerK** [comment]
I'm currently playing with
- https://github.com/hvanbakel/CsprojToVs2017
  - `dotnet migrate-2019 migrate -n -t net45 -t netcorapp3.1 -t net5.0` myProject.csproj
  seems todo a gut job, i will testing more.
- https://docs.microsoft.com/en-Us/dotnet/core/porting/upgrade-assistant-overview

to get experience with upgrating ;-) I have some projects in the queue to convert.

**Comment by Aldaviva** [comment]
I'm now attempting to use mwinapi from a .NET 6 project, so I am trying my hand at converting mwinapi to an SDK-style `.csproj` which multi-targets both .NET Framework 4 and .NET Core 3.1. So far I am able to successfully run a simple `SystemWindow.FilterToplevelWindows` using a local `.nupkg` file.

**Comment by Aldaviva** [comment]
I have published [mwinapi 0.3.0.5](https://www.nuget.org/packages/mwinapi/0.3.0.5) to NuGet Gallery with the .NET Framework 4 and .NET Core 3.1 multi-targeting. It seems to work for me, although I haven't tested it extensively.
